### ISSUE TITLE
Implement ControlValueAccessor interface in order to be able to use in in reactive forms

### ISSUE THREAD

**Issue by gschafra** [julianpoemp/ngx-jodit#25]
Currently, the component is implemented as "simple" component with 2-way value binding. For an (easier) integration in reactive forms, it would be nice, if the component implements the `ControlValueAccessor` interface.

Please answer additionally:

- Is this issue related to ngx-jodit or ngx-jodit-pro? 
  Relates to both
- What version of the package?
  `^4`
- What Angular version are you using?
  `^16`

**Comment by julianpoemp** [comment]
@gschafra thanks for the feature request! I'll have a look at it asap.

**Comment by julianpoemp** [comment]
@gschafra I just released ngx-jodit v3.1.0 with reactive forms support 🎉

**Comment by gschafra** [comment]
Great, thanks a lot. I'll try to test it today or next monday at latest.

**Comment by julianpoemp** [comment]
@gschafra great! After you confirmed it's working I'll release the feature for v2, too.

**Comment by gschafra** [comment]
@julianpoemp Unfortunately, it seems not to work, content shows always null, even with the value available:

![image](https://github.com/julianpoemp/ngx-jodit/assets/3999582/f086a99a-f09a-4864-ba0b-61462acda561)

I'll took a look into it next week, o.k.? Since I've already had a lot to do with ControlAccessor implementations, I think I'm able to contribute via PR. Keep in mind, that it is used in reactive forms this way (in contrast to you updated docs):

```html
<form [formGroup]="profileForm">
  <label for="info">Info: </label>
  <ngx-jodit id="info" type="text" [formControlName]="info" [options]="options"></ngx-jodit>
</form>
```
with a `profileForm` of type `FormGroup` defined in the corresponding component class (see https://angular.io/guide/reactive-forms#grouping-form-controls)

**Comment by julianpoemp** [comment]
Thanks for the feedback. Because I didn't use reactive forms for now I just tested the template-driven approach. Seems there is more work to do to get it working.

PR are highly appreciated :)

**Comment by gschafra** [comment]
I have identified the problem. It is a timing problem which apparently only occurs in the context of the reactive form (lifecycle). Value change is not reached initially here, since (optinal) jodit `ViewChild` is not (yet) available on `writeValue`:
https://github.com/julianpoemp/ngx-jodit/blob/32923e79d4728f367de453ede4be7dc437690dc7/libs/ngx-jodit/src/lib/ngx-jodit.component.ts#L34-L35
https://github.com/julianpoemp/ngx-jodit/blob/32923e79d4728f367de453ede4be7dc437690dc7/libs/ngx-jodit/src/lib/ngx-jodit.component.ts#L184-L190
One could make the `ViewChild` static, which implies the editor always to be present. Or do some RxJS pipline magic to react on the initialization :wink: . I've done the latter and will make it available as a PR straight away. But prepare yourself for something, I changed a lot in favour of, IMHO, a better "reactivity" :stuck_out_tongue_winking_eye: . I've also updated the demo and README, adding reactive form demo/docu.
